### PR TITLE
Remove BT_COEXIST option, it's garbage.

### PR DESCRIPTION
**Pull request by yagoplx**
See https://github.com/lwfinger/rtl8192ee/issues/11

### CODE REVIEW

**Comment by yagoplx** [comment]
I found out a day later that the interface will just drop packets at random, and this is not related to bluetooth interference. I am royally tired of this adapter which happens to be quite good (at least for its time) but was undermined by Realtek's shitty code, both in this rotting carcass of a multi-driver-treebomination and in the rtlwifi kernel branch.

At this point this "build my own router" project has been frustrating to no end all because of this chip. I now accept that I threw money in the trash. I'm not giving up on my project, though, just going to buy an ath9k (TL-WDN4800) card I found with a seller in my region. And some high gain antennas to have the 2.4G signal penetrate through my massive ancient walls. Maybe only then I will have some nice and strong server-powered WiFi.

If you have any advice about wireless and Linux, let me know.

Thanks for keeping these drivers around. I hope you enjoy maintaining these - as for me, the user and tester, it was not enjoyable. And this "driver vs. me" fight taught me nothing about the subject I wanted to learn about.  
TL;DR: Don't buy realtek
